### PR TITLE
Emit Type Declarations

### PR DESCRIPTION
**Pull request by charlie-tf**
- removing `emitDeclarationOnly` seems to have broken the type-
  checking in terraware-web
- add back to tsconfig.json
- tested with `yarn ts` in terraware-web locally

image

### CODE REVIEW

**Comment by karthikbtf** [comment]
I believe running `yarn test` introduced the previous changes.. might be good to run that just in case.
guessing we need a new distribution / version

**Comment by charlie-tf** [comment]
> I believe running `yarn test` introduced the previous changes.. might be good to run that just in case. guessing we need a new distribution / version

`yarn test` still works. Thanks.

**Review by karthikbtf — APPROVED** [review]
You may need to bump the version so we have a new distribution with .d.ts files

**Comment by charlie-tf** [comment]
> You may need to bump the version so we have a new distribution with .d.ts files

@karthikbtf Okay, I'll bump this one to 2.0.61.

You'll want to bump the version in https://github.com/terraware/web-components/pull/237 to 2.0.62 (I still need to figure out that auto-versioning thing 😅)

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
   "scripts": {
     "start": "start-storybook",
     "build": "build-storybook -c .storybook -o public",
-    "test": "react-scripts test",
+    "test": "cp tsconfig.json tsconfig.json.bak && react-scripts test && cp tsconfig.json.bak tsconfig.json && rm tsconfig.json.bak",
```
Comment (karthikbtf) [inline package.json:41]: Optionally use ‘mv’ instead of ‘cp’ and ‘rm’ after the test run

Comment (charlie-tf) [reply package.json:41]: oh, good call. that saves a command.

**Comment by karthikbtf** [comment]
> > You may need to bump the version so we have a new distribution with .d.ts files
> 
> @karthikbtf Okay, I'll bump this one to 2.0.61.
> 
> You'll want to bump the version in #237 to 2.0.62 (I still need to figure out that auto-versioning thing 😅)

Yes 👍

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**